src/System.Collections/tests/ILCConfigurations.rd.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Collections.Tests">
+    <Assembly Name="*Application*" DoNotInline="true" DoNotOptimize="true" />
+    <Assembly Name="System.Collections" DoNotInline="true" DoNotOptimize="true" />
```

Comment:
Is there an issue tracking this?  We don't want to actually ship this unoptimized, so it'd be good to have a comment in here mapping back to the issue so that we'll clean it up.